docker/opensource-docker/conf/core-site.xml

```diff
@@ -12,7 +13,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+-->
```

Comment:
Where did you copy this licence header? I see a different licence header in the default `core-site.xml` in hadoop 2.7.0.